manual/src/cmds/ocamldoc.etex

```diff
@@ -207,6 +207,10 @@ They have the same meaning as for the "ocamlc" and "ocamlopt" commands.
 Add \var{directory} to the list of directories search for compiled
 interface files (".cmi" files).
 
+\item["-H" \var{directory}]
+Like "-I", but the "-H" directories are searched last and the program may
+not directly refer to the modules included this way.
```

Comment:
The wording here is not great, but nobody cares because these are just `ocamldoc` options that duplicate compiler options. The current approach (adding meh formulations to a manpage no one reads anymore) is acceptable, but in the medium term it would be nice to get rid of these redundant option descriptions. ( @Octachron may know what would be a good way to do this.)

Reply:
My general plan for this issue would be to have a unique source of truth for all flags and their documentation and generate all formats and slices from this unique input.